qiita_db/support_files/patches/21.sql

```diff
@@ -0,0 +1,14 @@
+-- March 28, 2015
+-- Add default analyses for all existing users
+DO $do$
```

Comment:
I really like this patch

I think we can potentially use some of the SQL tricks to reduce some of the code in the qiita_db objects and we will improve the performance/quality of our code.